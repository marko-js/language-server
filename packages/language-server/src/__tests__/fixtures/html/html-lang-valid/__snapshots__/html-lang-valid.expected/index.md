## Diagnostics
### Ln 1, Col 2
```marko
> 1 | <html lang="35">
    |  ^^^^ Fix all of the following:
  Value of lang attribute not included in the list of valid languages
  2 |   <head>
  3 |     <title>Language 35</title>
  4 |   </head>
```

