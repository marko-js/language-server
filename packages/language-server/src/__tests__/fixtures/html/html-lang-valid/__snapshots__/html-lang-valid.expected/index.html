<html data-marko-node-id="0" lang="35">
  <head data-marko-node-id="1">
    <title data-marko-node-id="2">Language 35</title>
  </head>
</html>